raiden_contracts/data/gas.json

```diff
@@ -4,18 +4,18 @@
     "CustomToken.transfer": 36123,
     "CustomToken.transferFrom": 29557,
     "MonitoringService.claimReward": 41931,
-    "MonitoringService.monitor": 207277,
-    "OneToN.claim": 92769,
-    "SecretRegistry.registerSecret": 46828,
+    "MonitoringService.monitor": 207422,
+    "OneToN.claim": 92705,
+    "SecretRegistry.registerSecret": 46892,
     "TokenNetwork DEPLOYMENT": 4277369,
-    "TokenNetwork.closeChannel": 111236,
-    "TokenNetwork.openChannel": 97691,
+    "TokenNetwork.closeChannel": 111155,
+    "TokenNetwork.openChannel": 97745,
     "TokenNetwork.setTotalDeposit": 44909,
-    "TokenNetwork.setTotalWithdraw": 103225,
+    "TokenNetwork.setTotalWithdraw": 103306,
     "TokenNetwork.settleChannel": 108229,
     "TokenNetwork.unlock 1 locks": 31549,
-    "TokenNetwork.unlock 6 locks": 58585,
-    "TokenNetwork.updateNonClosingBalanceProof": 93769,
+    "TokenNetwork.unlock 6 locks": 58649,
+    "TokenNetwork.updateNonClosingBalanceProof": 93786,
```

Comment:
Why do so many gas costs change?

Reply:
That's a mystery https://github.com/raiden-network/raiden-contracts/issues/910 .